packages/eventual-send/test/test-ses-ava-reject-deep-stacks.js

```diff
@@ -0,0 +1,69 @@
+import { wrapTest } from '@agoric/ses-ava';
+import '@agoric/install-ses';
+import rawTest from 'ava';
+// import { E } from './get-hp';
+
+// This file is a variation of the `test-ses-ava-reject` from the
+// `@agoric/ses-ava` package. The difference is that this test
+// uses `E.when` rather than `then` to demonstrate deep stacks.
+
+const test = wrapTest(rawTest);
+
+test('ses-ava reject console output', t => {
+  t.assert(true);
+  // Uncomment this to see something like the text in the extended comment below
+
+  /*
+  return E.when(Promise.resolve(null), v1 =>
+    E.when(v1, v2 =>
+      E.when(v2, _ => {
+        assert.typeof(88, 'string', assert.details`msg ${'NOTICE ME'}`);
+      }),
+    ),
+  );
+  */
```

Comment:
Commented out tests seem really awkward.

Perhaps use `t.skip`?

Or perhaps the pattern from `test-deep-stacks.js`?
https://github.com/Agoric/agoric-sdk/blob/bae5b0d7ceb5c29007f03d02e68fae30bf78896b/packages/eventual-send/test/test-deep-stacks.js#L15-L18

I played around with it and came up with...

```suggestion

  const q = E.when(Promise.resolve(null), v1 =>
    E.when(v1, v2 =>
      E.when(v2, _ => {
        assert.typeof(88, 'string', assert.details`msg ${'NOTICE ME'}`);
      }),
    ),
  );
  return q.catch(reason => {
    t.assert(reason instanceof Error);
    t.notRegex(reason.message, /NOTICE ME/, 'message detail hidden');
    console.log('expected failure', reason);
  });
```

Reply:
meanwhile [our tests aren't supposed to console.log](https://github.com/Agoric/agoric-sdk/wiki/agoric-sdk-unit-testing#patterns). `t.log` would be silent unless `ava -v` were used...

```js
    t.log('expected failure', reason);
```

but it's not integrated with the SES console, so we get the boring...

```
  ✔ ses-ava-reject-deep-stacks › ses-ava reject console output
    ℹ expected failure TypeError {
        message: 'msg (a string)',
      }
```

I'm raising an issue on `ses-ava` about `t.log` support.

Reply:
I do not understand this. Let's discuss soon.